docs/source/standard.md

```diff
@@ -223,33 +223,87 @@ The ability of the TRE organisation to manage data assets and ensure information
 
 The ability of the TRE organisation to manage how and where data is stored, how it moves, changes and is removed.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-data-lifecycle-management
+
+* - Statement
+  - Guidance
+* - A TRE must have a data ingress process which enforces information governance rules/processes.
+  - The data ingress process needs to ensure that information governance is correctly followed.
+    in particular, it should require that an ingress request has been approved by all required par ties.
+* - A TRE must have a data egress process which enforces information governance rules/processes.
+  - The data egress process needs to ensure that information governance requirements are adhered to.
+    In particular, it should require that an egress request has been approved by all required parties.
+* - A TRE could require independent approval for data ingress and egress events.
+  - A person independent to a project, for example a TRE admin or dedicated referee, could check ingress and egress requests to ensure information governance processes are correctly followed in the absence of an automated process.
```

Comment:
> in the absence of an automated process

This reads like there's an expectation of an automated process, does any TRE currently have one?

Also (for the future)- we need to agree what "project" means. E.g. is the data provider part of the project? If so they should be allowed to authorise egress, even though they're not independent from the project.

Reply:
Our data handling procedures assume that:
- the TRE hosting organisation (Turing in our case)
- the project team (may or may not be Turing employees)
- each of the data providers
are stakeholders in the data analysis and should have a say in what gets ingressed and egressed.

Reply:
So, for now, are we all OK with this if

1. Reference to an automated process is removed
2. We leave 'project' to be defined or adjusted later
3. Add some context to _why_ the TRE organisation may want to (or need to) have input on egress approval?

Reply:
Yes, I think focussing on the requirement (there should be _a_ process) and the main reasons/context for _why_ a process is required makes sense.